vHunter/db/nvdapi_adapter.py

```diff
@@ -0,0 +1,57 @@
+import aiohttp
+import async_timeout
+import logging
+from pprint import pformat
+
+from vHunter.db import BasicDbAdapter
+from vHunter.utils import Config
+
+
+class NvdapiAdapter(BasicDbAdapter):
+    def __init__(self, min_score=5.0, max_pages=5):
+        config = Config()['nvdapi']
+        self.api_url = config['url']
+        self.api_port = config['port']
+        self.api_resource = config['resource']
+        self.min_score = min_score
+        self.max_pages = max_pages
+
+    def prepare_request(self):
+        return "{domain}:{port}{resource}".format(
+            domain=self.api_url,
+            port=self.api_port,
+            resource=self.api_resource
+        )
+
+    def prepare_payload(self, thing, version=None, vendor=None):
+        payload = {"product": thing}
+        if version is not None:
+            payload["version"] = version
```

Comment:
inline version would be more elegant:
`payload["version"] = version if version is not None`